src/models/teamModel.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+
+const teamSchema = new Schema({
+  name: {
+    type: String,
+    required: true,
+  },
+  users: [
+    {
+      type: Schema.Types.ObjectId,
+      ref: 'User',
+    },
+  ],
+});
```

Comment:
Could you add timestamp support please ?
```suggestion
},
{
    timestamps: true,
 });
```

Reply:
Is it an oversight or a wish not to add this part? :)